test/unit/errors/error-collector.test.js

```diff
@@ -1675,6 +1676,30 @@ test('Errors', async (t) => {
     })
   })
 
+  await t.test('error events', async (t) => {
```

Comment:
there's already a suite below this named `error events`, any reason not to just add test to that one?

Reply:
Slight oversight. I'll move it.